xmodule/vertical_block.py

```diff
@@ -250,7 +250,17 @@ def definition_from_xml(cls, xml_object, system):
         children = []
         for child in xml_object:
             try:
-                child_block = system.process_xml(etree.tostring(child, encoding='unicode'))
+                def_id = None
+                if is_pointer_tag(child):
+                    id_generator = system.id_generator
+                    url_name = child.get('url_name')
+                    block_type = child.tag
+                    def_id = id_generator.create_definition(block_type, url_name)
+
+                    child, _ = cls.load_definition_xml(child, system, def_id)
+                    child.set('pre_loaded', 'True')
```

Comment:
As per discussion it's better to send the boolean in kwargs rather than adding a flag in the xml and removing it later.